pkg/scaffold/operator.go

```diff
@@ -59,6 +59,14 @@ spec:
           command:
           - {{.ProjectName}}
           imagePullPolicy: Always
+          readinessProbe:
+            exec:
+              command:
+                - stat
+                - /tmp/operator-sdk-leader
+            initialDelaySeconds: 4
```

Comment:
Can I ask about the numbers, what is the reason behind 4 seconds delay?

Reply:
Small enough to not cause a lot of delay, but large enough to give the process a chance to start. Otherwise it's just a feeling. :) It's hard to pick these numbers well. It could probably be 2 seconds or 6 seconds with just as much success. I'm certainly open to any suggestions.

Reply:
My only concern was that the delay might not be enough. What happens then?

Reply:
It will try again 10 seconds later, so I don't see an opportunity for it to cause a problem.